pandas/tests/arrays/sparse/test_arithmetics.py

```diff
@@ -24,7 +24,7 @@ def mix(request):
 class TestSparseArrayArithmetics:
 
     _base = np.array
-    _klass = pd.SparseArray
+    _klass = pd.arrays.SparseArray
```

Comment:
can we just import SparseArray at the top?

Reply:
done